runtime/j9vm/jvm.c

```diff
@@ -5491,10 +5491,18 @@ JVM_ActiveProcessorCount(void)
 J9Class* java_lang_Class_vmRef(JNIEnv* env, jobject clazz);
 
 /**
- * JVM_GetClassName
+ * JVM_GetClassName / JVM_InitClassName
+ *
+ * The name was changed from JVM_GetClassName to JVM_InitClassName
+ * in interface version 6.
  */
 jstring JNICALL
-JVM_GetClassName(JNIEnv *env, jclass theClass)
+#if (JAVA_SPEC_VERSION < 11) || (JAVA_SPEC_VERSION == 12)
+JVM_GetClassName
+#else /* (JAVA_SPEC_VERSION < 11) || (JAVA_SPEC_VERSION == 12) */
+JVM_InitClassName
```

Comment:
How do we know the implementation for JVM_InitClassName is the same as for JVM_GetClassName?

Reply:
Class.java seems to use the methods the same way: I think openjdk just changed the name not its behavior.

Reply:
Note that since this isn't used, a stub that asserts would be fine.